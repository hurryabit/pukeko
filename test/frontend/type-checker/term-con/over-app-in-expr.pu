data T a = C a
f :: a -> T a
f x = C x x
