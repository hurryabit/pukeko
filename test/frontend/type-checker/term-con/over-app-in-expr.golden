test/frontend/type-checker/term-con/over-app-in-expr.pu:3:7: mismatching types T and (->) _1
